pyartcd/pyartcd/pipelines/prepare_release.py

```
@@ -983,6 +987,55 @@ async def remove_builds_all(self, advisory_id):
+            policy_data = yaml.load(policy_content)
+            actual_policy = policy_data.get("spec", {}).get("policy", "")
+
+            # Validate the policy
+            if actual_policy != expected_policy:
+                error_msg = (
+                    f"Advisory stage policy mismatch for {'EC' if is_ec_release else 'non-EC'} release. "
+                    f"Expected '{expected_policy}', but found '{actual_policy}' in {policy_filename}"
+                )
+                _LOGGER.error(error_msg)
+                await self._slack_client.say_in_thread(f":warning: {error_msg}")
+                raise ValueError(error_msg)
```

Ruff (0.14.10)

Abstract `raise` to an inner function

(TRY301)